QuasarHelios2015/src/org/team1540/quasarhelios/AutoLoader.java

```diff
@@ -67,6 +75,12 @@ public void autonomousMain() throws AutonomousModeOverException, InterruptedExce
             running.set(false);
         }
     }
+    
+    private void setClampHeight(float value) throws AutonomousModeOverException, InterruptedException {
+        Clamp.mode.set(Clamp.MODE_HEIGHT);
+        Clamp.height.set(value);
+        waitUntil(FloatMixing.floatIsInRange(Clamp.heightReadout, value - Clamp.heightPadding.get(), value + Clamp.heightPadding.get()));
```

Comment:
If you merge auto-stacker into this branch, you can just use ``Clamp.atDesiredHeight`` instead of ``floatIsInRange``.